catalogue/webapp/pages/concept.tsx

```diff
@@ -128,81 +109,75 @@ export const ConceptPage: NextPage<Props> = ({
   const hasImagesTabs = !!(imagesBy?.totalResults && imagesAbout?.totalResults);
 
   return (
-    // TODO fill meta information; who decides this?
+    // TODO fill meta information; wait for confirmation on description
     <CataloguePageLayout
       title={conceptResponse.label}
-      description="<TBC>"
+      description={`Find out more about ${conceptResponse.label} by browsing related works and images from Wellcome Collection.`}
       url={{ pathname: `/concepts/${conceptResponse.id}`, query: {} }}
       openGraphType="website"
       siteSection="collections"
       jsonLd={{ '@type': 'WebPage' }}
       hideNewsletterPromo={true}
     >
-      <ConceptHero
-        v={{ size: 'xl', properties: ['padding-top', 'padding-bottom'] }}
-      >
+      <ConceptHero>
         <div className="container">
-          <ConceptDescription>
-            <h1 className="font-intb">{conceptResponse.label}</h1>
-            {/* TODO dynamise */}
-            {FAKE_DATA.description && (
-              <p className="font-size-5">{FAKE_DATA.description}</p>
-            )}
-            {/* TODO dynamise */}
-            {FAKE_DATA.urls?.length > 0 &&
-              FAKE_DATA.urls.map(link => {
-                /* TODO Could they be internal links? Check if external to display arrow, decide on rel. */
-                return (
-                  <a
-                    key={link.url}
-                    href={link.url}
-                    rel="nofollow"
-                    className={font('intr', 6)}
-                  >
-                    {link.label} ↗
-                  </a>
-                );
-              })}
-          </ConceptDescription>
+          <TypeLabel>{conceptResponse.type}</TypeLabel>
+          <Space v={{ size: 's', properties: ['margin-top', 'margin-bottom'] }}>
+            {/* TODO only getting Subjects, confirm if it will change or not */}
+            <HeroTitle>{conceptResponse.label}</HeroTitle>
+            {/* TODO get copy from Jonathan */}
+            <ConceptDescription className="font-size-5">
```

Comment:
I think using the `font` function would be nicer (and think we've tended to try to use it elsewhere), because it's typed and we could change the underlying api without having to through and change classnames after.

Reply:
The type is a good reason, but I didn't like that it would add the `font-intr` class to it even though it's already using Inter, I thought that was the reasoning between using it or not?

Reply:
I think we've been happy to re-add an already-applied property as a necessary evil and avoided using the classnames ourselves directly almost everywhere. I _think_.

Reply:
maybe we could change the `font` function to work without the need for a family name